carta/structs.py

```diff
@@ -0,0 +1,39 @@
+"""This module provides a collection of helper objects for grouping related values together."""
+
+import posixpath
+
+from .validation import validate, String, Constant
+from .constants import Polarization
+
+
+class StokesImage:
+    '''An object which groups information about an image file to be used as a component in a Stokes hypercube.
+
+    Parameters
+    ----------
+    stokes : :obj:`carta.constants.Polarization`
+        The Stokes type to specify. Must be a member of :obj:`carta.constants.Polarization`
+    path : str
+        The path to the image file.
+    hdu : str
```

Comment:
These descriptions should be consistent with what would be filled in by the validation decorator (it won't be filled in here because the decorator is on the `__init__` method and the docstring is on the whole class).

So it should be
```
a member of :obj:`carta.constants.Polarization`
```

and `string` for the other two.

Have you been able to build the docs locally?

Reply:
Done in 27031ca!
Yes, I am able to build the docs locally~